swap-logic/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "swap-logic"
+version = "0.1.0"
+authors = ["Bryan RHEE <bryan.rhee@hdac.io>"]
+edition = "2018"
+
+[lib]
+crate-type = ["lib", "cdylib"]
```

Comment:
Do we need to support also cdylib?
If we use this crate only as lib, then we also don't need to commit Cargo.lock.

Reply:
https://github.com/hdac-io/swap/pull/1/commits/20729abfb6211eece9df8ae624629722b09c6b6e